### DISCUSSION TITLE
Chrome not browsing

### DISCUSSION THREAD

**Discussion by Remraf1** [cmu-sei/GHOSTS#98] [General]
I'm running Chrome 96.0.4664.110 on my windows client, my ChromeDriver version is 96.0.4664.45 (closest I could get).

When I start the Ghost client my Chrome window opens, but the URL displayed is "data:,".  I'm using the Browse command in the timeline file with a verified valid URL.

In my app.log file I see this error:
     Ghosts.Client.Handlers.BrowserChrome..ctor|System.InvalidOperationException: Session not created
     from disconnected: Unable to receive message from renderer

Wondering if it's the Chrome version I'm using or maybe I need a newer ChromeDriver?  All the other automated tasks are working, just gotta get the browsing to work.

Unable to try IE or Firefox, the client only has Chrome and Edge.

**Comment by sei-dupdyke** [comment]
Probably a chrome driver mismatch with your version of chrome - you can match up the version you need on this page: https://chromedriver.chromium.org/downloads - just swap out your new version with the version that is in the ghosts application folder.

**Reply by Remraf1** [reply]
According to that page the Chrome version and Chrome driver versions match.  Going to try upgrading both to newer versions, mine are old anyway lol.

**Reply by sei-dupdyke** [reply]
Another thing you might try is to fire up Chrome and let it set. Now fire up chromedriver and see what its logs say. When you run it directly, it outputs logs to the terminal window.

**Reply by Remraf1** [reply]
Upgrading to 103 for both, no change.  I launched Chrome, then ChromeDriver and it seems ok, though it does say "Only local connections allowed".  Is that normal?

**Reply by Remraf1** [reply]
Fixed it!  Dang group policy setting on my client was killing it.
Computer Configuration > Administrative Templates > Google > Google Chrome > "Control where developer tools can be used"

Thank you as always for the help Dustin!

**Reply by sei-dupdyke** [reply]
Awesome! That is a new one. Hopefully, your investigation helps others in the future. Thanks for being patient and diving into that one!
